docs/best_practices/time_series.rst

```diff
@@ -17,76 +18,125 @@ Keep in mind that the dimensions are reversed in MatNWB, so in memory in MatNWB
+    with the ``starting_time`` incremented appropriately.
+    2. Even if the sampling rate is constant within each segment, a single ``TimeSeries`` can be written using the 
+    ``timestamps`` vector to appropriately indicate the gaps between segments.
 
 Check function: :py:meth:`~nwbinspector.checks.time_series.check_regular_timestamps`
 
 
 
+.. _best_practice_chunk_data:
 
-Unassigned
+Chunk Data
 ~~~~~~~~~~
 
-Use chunking to optimize reading of large data for your use case. By default, when using the HDF5 backend, TimeSeries data are stored on disk in C-ordering.
-This means that if “data” dataset of a TimeSeries has multiple dimensions, then all data from a single timestamp are stored contiguously on disk, then data
-from the next timestamp are stored contiguously. This storage scheme may be optimal for certain uses, such as slicing TimeSeries by time; however, it may be
-sub-optimal for other uses, such as reading data from all timestamps for a particular value in the second or third dimension. Data writers can optimize the
-storage of large data arrays for particular uses by using chunking and compression. For more information about chunking and compression, consult the PyNWB
-documentation and MatNWB documentation.
+Use chunking to optimize reading of large data for your use case.
+
+By default, when using the HDF5 backend, TimeSeries data are stored on disk in column-based ordering.
+
+This means that if the `data` of a TimeSeries has multiple dimensions, then all data from a single timestamp are stored
+contiguously on disk, followed by the next timestamp, and so on.
+
+This storage scheme may be optimal for certain uses, such as slicing TimeSeries by time; however, it may be sub-optimal
+for other uses, such as reading data from all timestamps for a particular value in the second or third dimension.
+
+This is especially important when writing NWBFiles that are intended to be uploaded to the 
+:dandi-archive:`DANDI Archive <>` for storage, sharing, and publication.
+
+For more information about how to enable chunking and compression on your data, consult the 
+:pynwb-docs:`PyNWB tutorial <tutorials/advanced_io/h5dataio.html#chunking>` or the
+`MatNWB instructions <https://neurodatawithoutborders.github.io/matnwb/tutorials/html/dataPipe.html#2>`_.
+
+
+.. _best_practice_large_dataset_compression:
+
+Compress Data
+~~~~~~~~~~~~~
+
+Data writers can optimize the storage of large data arrays for particular uses by using compression applied to each 
+chunk individually.
+
+This is especially important when writing NWBFiles that are intended to be uploaded to the 
+:dandi-archive:`DANDI Archive <>` for storage, sharing, and publication.
+
+For more information about how to enable compression on your data, consult the 
+:pynwb-docs:`PyNWB tutorial <tutorials/advanced_io/h5dataio.html#compression-and-other-i-o-filters>` or the
+`MatNWB instructions <https://neurodatawithoutborders.github.io/matnwb/tutorials/html/dataPipe.html#2>`_
```

Comment:
This is the best I could find for the MatNWB side; is there something newer? I notice this is under the older `github.io`